ledfx/core.py

```
@@ -177,14 +177,32 @@ def handle_base_configuration_update(self, event):
+
+        This centralizes policy decisions so API/config/server hooks only need
+        to signal *when* to re-check, not duplicate *what* to do.
+        """
+        if self.config.get("sendspin_always_on", True):
+            _LOGGER.debug(
+                "sendspin reconcile (%s): always-on enabled, checking eager start.",
+                trigger,
+            )
+            try:
+                sendspin_eager_start(self)
+            except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)